src/DataDictionary/graph/GraphDrawer/GraphDrawer.test.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { mount } from 'enzyme';
-import GraphDrawer from '../GraphDrawer/GraphDrawer';
+import GraphDrawer from './GraphDrawer';
```

Comment:
Why this change? Has the location of GraphDrawer moved?

Reply:
no, they are the same, both of those files are in `/GraphDrawer`

Reply:
ahhh i see hahaha